src/main/java/org/study/services/impl/AdministratorServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package org.study.services.impl;
+
+import org.apache.log4j.Logger;
+import org.study.DAO.AdministratorDAO;
+import org.study.factories.DAOFactory;
+import org.study.factories.DAOType;
+import org.study.models.Administrator;
+import org.study.services.AdministratorService;
+
+public class AdministratorServiceImpl implements AdministratorService {
+    /**
+     * create instance of DAO interface with type ADMINISTRATOR
+     */
+    private static final AdministratorDAO INSTANCE =
```

Comment:
rename at least to dao
Not clear what means Instance when read code, have to scroll and search variable definition

Reply:
Will be fix in next MR